applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -110,6 +110,7 @@ def AddVariables(self):
         # Kratos variables
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.FLAG_VARIABLE) # Required for variational_distance_process
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISTANCE)
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)
```

Comment:
Do we need it as historical variable?

Reply:
otherwise the REACTION variable will not be recognized by the solver hence aborting.
in model_part.h, I did not find any function that will just initialize the variable.